applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c

```diff
@@ -608,10 +535,21 @@ static int zbus_subscribers_create(void)
 	return 0;
 }
 
+uint8_t stream_state_get(void)
+{
+	return strm_state;
+}
+
+void streamctrl_send(void const *const data, size_t size, uint8_t num_ch)
+{
+	ARG_UNUSED(data);
+	ARG_UNUSED(size);
```

Comment:
Should we add a WRN/ERR in here for pointing out this is not used?

Reply:
Added a LOG_WRN()